server/src/main/java/org/opensearch/index/shard/IndexShard.java

```diff
@@ -1355,23 +1393,28 @@ public Engine.DeleteResult applyDeleteOperationOnReplica(long seqNo, long opPrim
                 version,
                 null,
                 Engine.Operation.Origin.REPLICA,
-                System.nanoTime(),
+                threadPool.absoluteTimeInMillis(),
```

Comment:
You definitely cannot replace a `nanoTime()` call with a call that returns milliseconds. Why did you change this code?